omod/src/main/resources/apps/adminui_configuremetadata_admin_links_extension.json

```diff
@@ -47,6 +47,18 @@
         }
     },
 
+    {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.visits",
+        "extensionPointId": "${project.parent.groupId}.${project.parent.artifactId}.adminLinks",
+        "type": "link",
+        "label": "adminui.manageVisitTypes.title",
+        "url": "adminui/metadata/visits/visittypes/manageVisitTypes.page",
```

Comment:
It's better to use maven placeholders for the moduleId

Reply:
What do you mean by maven placeholders for this line 55? I couldn't understand the placeholder usage here

Reply:
Replace `adminui` with `${project.parent.artifactId}`

Reply:
It's not necessary but I personally  prefer to do it that way, in the event the module has to be renamed or the code has to be moved to another project it's future proof.